src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Validation/InformationScreen/ValidationInformationViewModel.swift

```diff
@@ -0,0 +1,44 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+struct ValidationInformationViewModel {
+
+	// MARK: - Internal
+
+	var dynamicTableViewModel: DynamicTableViewModel {
+		DynamicTableViewModel([
+
+			// Illustration with information text and bullet icons with text
+			.section(
+				header:
+					.image(
+						UIImage(
+							imageLiteralResourceName: "Illu_Europe_Card"
+						),
+						title: AppStrings.HealthCertificate.Validation.Info.title,
+						accessibilityLabel: AppStrings.HealthCertificate.Validation.Info.imageDescription,
+						accessibilityIdentifier: AccessibilityIdentifiers.HealthCertificate.Validation.Info.imageDescription,
+						height: 274.0,
+						accessibilityTraits: [.header, .image]
+					),
+				cells: [
+					.icon(
+						UIImage(imageLiteralResourceName: "Icons_Car"),
+						text: .string(AppStrings.HealthCertificate.Validation.Info.byCar),
+						alignment: .top
+					),
+					.icon(
+						UIImage(imageLiteralResourceName: "Icons_Plane"),
+						text: .string(AppStrings.HealthCertificate.Validation.Info.byPlane),
+						alignment: .top
+					)
+				]
```

Comment:
The spacing seems a bit off in the screenshot, maybe you can add a .space(...) on top